NMS-13338: Add DnsDetector docs (#3445)
OpenNMS/opennms: docs/modules/operation/pages/provisioning/detectors/DnsDetector.adoc

= DNS Service Detector

> This detector is build to test the availability of the _DNS service_ on remote IP interfaces.

Reviewer: ```suggestion
This detector lets you find DNS services for monitoring in your network.
```

> The detector tests the service availability by sending a DNS query for A resource record types against the DNS server to test.

Reviewer: ```suggestion
The DNS service is detected when the server responds to a DNS query for an A resource record.
You can customize the A record used in the query with the `lookup` configuration parameter.
```

= DNS Service Detector

> This detector is build to test the availability of the _DNS service_ on remote IP interfaces.

Reviewer 2: ```suggestion
This detector lets you find DNS services to monitor in your network.
```
Please verify this update is correct.

The detector tests the service availability by sending a DNS query for A resource record types against the DNS server to test.

> The detector can be simulated with the command line tool `host`:

Reviewer 2: ```suggestion
Use the command line tool `host` to simulate the detector:
```
